wp-content/themes/core/components/accordion/accordion.php

```diff
@@ -0,0 +1,55 @@
+<?php
+// phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
+$controller = \Tribe\Project\Templates\Components\accordion\Controller::factory( $args );
+?>
+<div
+	<?php echo $controller->container_classes(); ?>
+		role="tablist"
+		aria-multiselectable="true"
+	<?php echo $controller->container_attrs(); ?>
+>
+	<?php foreach ( $controller->rows as $key => $row ) { ?>
+	<article <?php echo $controller->row_classes(); ?>>
+		<<?php echo esc_attr( $controller->row_header_tag ); ?>>
+		<button
+				aria-controls="<?php echo esc_attr( $row[ 'content_id' ] ); ?>"
+				aria-expanded="false"
+				aria-selected="false"
+			<?php echo $controller->row_header_classes(); ?>
+				id="<?php echo esc_attr( $row[ 'header_id' ] ); ?>"
+				data-index="<?php echo absint( $key ); ?>"
```

Comment:
I think we can drop the data-index here & similarly, the `data-name`, `data-index`, and `data-livetext` attributes throughout this component as those are specifically for panel builder.

Reply:
Yep!